src/Sylius/Bundle/LocaleBundle/Resources/config/services.xml

```diff
@@ -51,7 +51,7 @@
         <service id="sylius.listener.request_locale_setter" class="Sylius\Bundle\LocaleBundle\Listener\RequestLocaleSetter">
             <argument type="service" id="sylius.context.locale" />
             <argument type="service" id="sylius.locale_provider" />
-            <tag name="kernel.event_listener" event="kernel.request" priority="4" />
+            <tag name="kernel.event_listener" event="kernel.request" priority="6" />
```

Comment:
Can we check if we need to touch this place anyway? maybe we can just lower value here: https://github.com/Sylius/Sylius/pull/12458/files#diff-9511fd2b1a63d14d03a33b1037a4766e7305ed03adf9b848cbd3f56fa1e403d3R44 and leave this place as it was

Reply:
seems like its working well just with priority 4 here